tests/test_cli.py

```diff
@@ -3,7 +3,7 @@
 from submitter.cli import main
 
 
-def test_cli_sample_data_loader(mocked_sqs):
+def test_cli_load_sample_data(caplog, mocked_sqs):
```

Comment:
Is caplog automatically passed in via the testing framework?

Reply:
Oh, yes but we don't need it now that we're not getting errors around logging in the cli. Will update.